examples/cfd/07_cavity_flow.ipynb

```diff
@@ -0,0 +1,700 @@
+{
```

Comment:
<p>How expensive would it be to run the 1024 x 1024 with devito? For CI may be too much but could be nice to run it once and save the values to put on the plot.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1502/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
Thanks, it seems quite expensive from my attempts at running 1024 x 1024. I'd have to use a smaller time-step (to account for the CFL condition) and thus more time steps are needed to reach steady state. Also more pressure equation iterations might be needed. But, if it's worthwhile, I could get it to work at some point.

Reply:
We could maybe leave that until implicit support is added? We'll need to update this notebook again at that stage anyway.

In any case, I've approved. Thanks again Kene.